app/views/admin/manage_user.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <h2><%= t ".manage_users" %></h2>
+  <ul class="users">
+    <%= paginate @manage_users, theme: "twitter-bootstrap-3" %>
+    <% @manage_users.each do |user| %>
+      <table class="table table-condensed">
+        <tbody>
+          <tr>
+            <td rowspan="3"><%= gravatar_for user, size: 50 %></td>
+            <td><%= link_to user.name, user %></td>
+            <td rowspan="3"><% if !current_user?(user) %>
```

Comment:
logic chỗ này là sao  nhỉ? Đây trong admin anh  tưởng nó có toàn quyền?

Reply:
Là khi admin đang đăng nhập thì không đc xóa chính mình ạ

Reply:
Nên check nó là admin nữa nhé, vì nếu 1 user khác dc xét quyền quản trị thì nó vẫn xóa dc admin à?